### PR TITLE
LPS-166934  [CP 2.0] Add Automated testcase

### PR DESCRIPTION
**Pull request by styller1**
https://issues.liferay.com/browse/LPS-166934

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

**Comment by styller1** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro [hunk 1]
```diff
@@ -175,6 +214,18 @@ definition {
 		}
 	}
 
+	macro searchKeys {
+		Click(locator1 = "CP2NavigationMenu#SEARCH_FILTER");
+
+		Type(
+			locator1 = "CP2NavigationMenu#SEARCH_FILTER",
+			value1 = "${searchinput}");
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:222]: Would you mind changing it to searchInput, please?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path [hunk 2]
```diff
@@ -95,6 +95,11 @@
 	<td>//a[contains(@class, 'font-weight-bold')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEARCH_FILTER</td>
+	<td>//div[contains(@class,'bg-neutral-1')]//div[@class="input-group-item"]//input[@placeholder="Search"]</td>
+	<td></td>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path:101]: without the "/div[contains(@class,'bg-neutral-1')]", does the locator work?

Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2NavigationMenu.path:101]: No, without that, the path finds more than 1 result

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro (hunk 1 above)
Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:222]: I will do it

**Comment by styller1** [comment]
ci:test:sf

**Pull request closed without merging**